api/src/test/java/org/pmiops/workbench/exfiltration/EgressEventServiceTest.java

```diff
@@ -186,7 +186,6 @@ public void testCreateEgressEventAlert() throws ApiException {
     egressEventService.handleEvent(event);
     verify(mockAlertApi).createAlert(alertRequestCaptor.capture());
     verify(egressEventAuditor).fireEgressEvent(event);
-    verify(mockTaskQueueService).pushEgressEventTask(anyLong());
```

Comment:
we should restore this line for the V2 test at line 213.

Reply:
Good catch - done